cluster/charts/rook-ceph-cluster/values.yaml

```diff
@@ -278,3 +288,103 @@ cephClusterSpec:
         disabled: false
       osd:
         disabled: false
+
+cephStorageSpec:
+  blockpools: []
```

Comment:
Instead of commenting all these out, what if we go ahead and create a pool, filesystem, and object store, and enable their storage classes in the default values.yaml? This will make it easier to see a working helm chart with most everything enabled. Of course if they don't want those services they could remove them. In the docs we would just need to be very clear about what is created by default.

Reply:
I've updated to include the examples by default, and added documentation to describe what the values are